spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/732/posts' do
+    it 'returns a 200 OK status' do
+      get '/users/732/posts'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the index template' do
+      get '/users/732/posts'
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'GET /users/732/posts/:id' do
```

Comment:
- [ ] **[OPTIONAL]** I would advise you to use a more descriptive text here to describe what you are testing.

  ```
  describe GET #show
  ```